app/templates/frameworks/submission_lots.html

```diff
@@ -34,34 +34,156 @@
-            {% endif %}
-          </div>
-        {% endif %}
+      {% if framework.family == 'g-cloud' %}
+        <div class="use-of-service-data">
+          {% if framework.status == 'pending' and not application_made %}
+            <p class="govuk-body">The services below were not submitted.</p>
+          {% else %}
+          <p class="govuk-body">The service information you provide in your application:</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>will be on your public service description page</li>
+            <li>will help buyers review and compare services</li>
+            <li>may be used as filters in search</li>
+          </ul>
+          {% endif %}
+        </div>
+      {% endif %}
+      {% if framework.status == 'open' %}
+        {{ govukButton({
+          "text": "Add a service",
+          "href": url_for('.choose_draft_service_lot', framework_slug=framework.slug)
+        })}}
+      {% endif %}
     </div>
   </div>
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds framework-lots-table">
-      {% with items = lots %}
-        {% include "toolkit/browse-list.html" %}
-      {% endwith %}
-<p class="govuk-body">&nbsp;</p>
-    <a class="govuk-link" href="{{ url_for('.framework_dashboard', framework_slug=framework.slug) }}">Back to {{ framework.name }} application</a>
+    <div class="govuk-grid-column-two-thirds">
+
+      {% if framework.status == 'open' %}
+        {% set ns = namespace(complete_drafts_rows = [], drafts_rows = [], head = []) %}
+
+        {% if framework.family == "g-cloud" %}
+
+          {% set head = [
+            {"text": "Name", "classes": "govuk-!-width-one-half"},
+            {"text": "Lot", "classes": "govuk-!-width-one-quarter"},
+            {"text": "", "classes": "govuk-!-width-one-quarter"}
+          ] %}
+
+          {% for draft in drafts|sort(attribute="serviceName") %}
+            {% set row = ns.drafts_rows.append(
+              [
+                {"text": draft["serviceName"]},
+                {"text": draft["lotName"]},
+                {
+                  "html": '<a href="' + url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft["id"]) + '">Edit</a>',
+                  "classes": "dm-action-link"
+                }
+              ]
+            )%}
+          {% endfor %}
+          
+          {% for draft in complete_drafts|sort(attribute="serviceName") %}
+            {% set row = ns.complete_drafts_rows.append(
+              [
+                {"text": draft["serviceName"]},
+                {"text": draft["lotName"]},
+                {
+                  "html": '<a href="' + url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft["id"]) + '">Edit</a>',
+                  "classes": "dm-action-link"
+                }
+              ]
+            )%}
+          {% endfor %}
+        {% else %}
```

Comment:
DOS lots user research studios and user research participants can have more than one service per lot, so we probably do want to show names for those services, otherwise we end up with a less than helpful table :(

![Table "Drafts (2)" with two rows, "User research studios" and "User research studios"](https://user-images.githubusercontent.com/503614/110341635-767bcb80-8022-11eb-9f1f-a6c0a0d4f1c8.png)

Reply:
Hmm, I missed this. It looks like User Research Participants has `"oneServiceLimit": True` - is that incorrect?

I'll add something in for user research studios - thanks for catching!

Reply:
Ah, I misremembered, I guess it is just user research studios.

Reply:
Yeah, the table thing is annoying, I think your solution is elegant though